content/4_Gold/Combinatorics.problems.json

```diff
@@ -118,6 +118,18 @@
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "ac-XorPyramid",
+      "name": "Xor Pyramid",
+      "url": "https://cses.fi/problemset/task/2419",
+      "source": "AC",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Combinatorics", "Bitwise Operators"],
```

Comment:
change bitwise operators to just bitwise. no other problems use this tag

Reply:
Done